lib/read/setup.js

```diff
@@ -86,39 +105,39 @@ class Setup {
         });
     }
 
-    async createTargetConnections(targets, cwd)
```

Comment:
We need to concept of `targets` in order to support annotations related to other VMs, or when there is a host and VM specific instructions. Make an issue/task to keep track of this missing functionality.

Reply:
Handling in #19